publish.sbt

```diff
@@ -0,0 +1,31 @@
+ThisBuild / organization := "io.github.conduktor"
+
+ThisBuild / scmInfo := Some(
+  ScmInfo(
+    url("https://github.com/conduktor/kafka-security-manager"),
+    "scm:git@github.com:conduktor/kafka-security-manager.git"
+  )
+)
+ThisBuild / developers := List(
```

Comment:
this doesn't seem to be in line with `build.sbt`. Is this information really needed to be duplicated?